server/src/main.rs

```diff
@@ -210,11 +212,11 @@ fn main() {
       // static resources
       .service(actix_files::Files::new("/static", front_end_dir()))
   })
-  .bind("0.0.0.0:8536")
+  .bind((settings.bind, settings.port))
   .unwrap()
   .start();
 
-  println!("Started http server: 0.0.0.0:8536");
+  println!("Started http server at {}:{}", settings.bind, settings.port);
   let _ = sys.run();
 }
 
```

Comment:
I was gonna suggest adding these to all the docker-compose files, env files, and the ansible config, but since we're going to be refactoring those into a config file soon to simplify, and you already have working defaults for these, don't worry about that.

Thanks!